### PR TITLE
First attempt at getting ensureSasl{Config,Users} working.

### PR DESCRIPTION
**Pull request by mgoulish**
Andy please take a look.
I have not yet fixed the issue where router pod does not come up when --enable-router-console flag present, but that happens with or without this code. See what you think...

### CODE REVIEW

File: client/van_router_create.go [hunk 1]
```diff
@@ -541,3 +551,75 @@ func (cli *VanClient) VanRouterCreate(ctx context.Context, options types.VanRout
 
 	return nil
 }
+
+func (cli *VanClient) ensureSaslConfig(owner *metav1.OwnerReference) error {
+	name := "skupper-sasl-config"
```
Comment (ajssmith) [inline client/van_router_create.go:556]: suggest moving to /api/types

File: client/van_router_create.go [hunk 2]
```diff
@@ -541,3 +551,75 @@ func (cli *VanClient) VanRouterCreate(ctx context.Context, options types.VanRout
...
+	_, err := cli.KubeClient.CoreV1().ConfigMaps(cli.Namespace).Get(name, metav1.GetOptions{})
+	if err == nil {
+		fmt.Println("sasl config already exists")
```
Comment (ajssmith) [inline client/van_router_create.go:559]: return as fmt.Errorf

File: client/van_router_create.go [hunk 3]
```diff
@@ -541,3 +551,75 @@ func (cli *VanClient) VanRouterCreate(ctx context.Context, options types.VanRout
...
+			return fmt.Errorf("Failed to create sasl config: ", err.Error())
+		}
+	} else {
+		return fmt.Errorf("Failed to check for sasl config: ", err.Error())
+	}
+	return nil
+}
+
+func (cli *VanClient) ensureSaslUsers(user string, password string, owner *metav1.OwnerReference) error {
+	name := "skupper-console-users"
```
Comment (ajssmith) [inline client/van_router_create.go:594]: as above, move types

File: client/van_router_create.go [hunk 4]
```diff
@@ -541,3 +551,75 @@ func (cli *VanClient) VanRouterCreate(ctx context.Context, options types.VanRout
...
+	_, err := cli.KubeClient.CoreV1().Secrets(cli.Namespace).Get(name, metav1.GetOptions{})
+	if err == nil {
+		fmt.Println("console users secret already exists")
```
Comment (ajssmith) [inline client/van_router_create.go:597]: return fmt.Errorf

**Comment by mgoulish** [comment]
Closing this PR in favor of new one to show up momentarily.

**Pull request closed without merging**